pkg/api/service/util.go

```diff
@@ -66,3 +69,58 @@ func GetLoadBalancerSourceRanges(service *api.Service) (netsets.IPNet, error) {
 	}
 	return ipnets, nil
 }
+
+// RequestsOnlyLocalTraffic checks if service requests OnlyLocal traffic.
+func RequestsOnlyLocalTraffic(service *api.Service) bool {
+	if service.Spec.Type != api.ServiceTypeLoadBalancer &&
+		service.Spec.Type != api.ServiceTypeNodePort {
+		return false
+	}
+	// First check the alpha annotation and then the beta. This is so existing
+	// Services continue to work till the user decides to transition to beta.
+	// If they transition to beta, there's no way to go back to alpha without
+	// rolling back the cluster.
+	for _, annotation := range []string{AlphaAnnotationExternalTraffic, BetaAnnotationExternalTraffic} {
```

Comment:
Shouldn't we drop the alpha annotations now?  I guess it doesn't hurt to leave them...